etc/upgrades/1696936385-refunds_enhancement/common.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "refunds_enhancement",
```

```suggestion
  "name": "refunds-enhancement",
```
for consistency with other upgrade names. Also, please replace `_` with `-` in folder name